lib/solidus_dev_support/testing_support/factories.rb

```diff
@@ -59,13 +59,13 @@ def self.load_for(*engines)
             else
               factories_file_or_folder
             end
-          end.map { |path| path.sub(/.rb\z/, '').to_s }
+          end
         end
 
         if using_factory_bot_definition_file_paths?
           FactoryBot.definition_file_paths = [
             Spree::TestingSupport::FactoryBot.definition_file_paths,
-            paths,
+            paths.map { |path| path.sub(/.rb\z/, '').to_s },
```

Comment:
Did you get an error with the previous code? I'm quite sure `require` works well with or without the extension, but I was curious about that